### ISSUE TITLE
Codec licenses disabled - RPi2

### ISSUE THREAD

**Issue by Dimensional** [maxnet/berryboot#231]
I had purchased the MPG2 and WVC1 codec licenses a few months back, and had used them before with BerryBoot, but something in the recent update has caused the same licenses to not be read by the systems. I tested to see if they were for another CPU serial, if I had entered them incorrectly, and even erased everything and installed Raspbian through NOOBS, and so far everything just points to BerryBoot being the problem.

I'm unsure what exactly the issue is, but all I know is that when I added the codec licenses to config.txt from the NOOBS Raspbian, it would read and enable the codecs, but using the Raspian that I downloaded through BerryBoot didn't work. Even taking the full Raspbian image and converting it with the custom distro instructions didn't make it work.

I'm not sure where to look to include in a bug report, however I do notice that when I used the 'vcgencmd get_config str' command, it returned different lines of text between BerryBoot and NOOBS versions. BerryBoot has device_tree, kernel, and ramfsfile. The NOOBS version had a blank device tree and the two decode_MPG2|WVC1 lines.

**Comment by maxnet** [comment]
![berryboot_menu2](https://cloud.githubusercontent.com/assets/1123701/11769883/00098492-a1f2-11e5-9480-105e67d5a416.png)

Make sure you use the config editor to add your keys.

And be careful where you add them.
If you have a Pi 2, do NOT add them to the [pi1] section.

**Comment by Dimensional** [comment]
That's what I used to enter the data. My custom images had it's own boot directory, which was odd given I only created an image of loop0p1 and ignored the boot partition. The provided images had it mounted automatically.

Edit: I see. Didn't think that would make a difference where it was added, even at the end of the file.

Edit2: Thank you. I moved it from the bottom of the file to just before the Pi1 statement, inside the Pi2 block, and it now works.
